gtsfm/pose_slam/pose_slam.py

```diff
@@ -0,0 +1,85 @@
+"""Pose SLAM initialization module from relative pose priors.
+
+Authors: Akshay Krishnan, Frank Dellaert
+"""
+from typing import Dict, List, Optional, Tuple
+
+import dask
+import gtsam
+from dask.delayed import Delayed
+from gtsam import Pose3
+
+import gtsfm.utils.logger as logger_utils
+from gtsfm.common.pose_prior import PosePrior
+from gtsfm.evaluation.metrics import GtsfmMetric, GtsfmMetricsGroup
+
+PRIOR_NOISE_SIGMAS = [0.001, 0.001, 0.001, 0.1, 0.1, 0.1]
```

Comment:
super strict on rotations?

Reply:
It is arbitrary. This prior will be satisfied exactly